OurUmbraco/Auth/UmbracoMemberAuthApiController.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Umbraco.Core.Models;
+using Umbraco.Web.WebApi;
+
+namespace OurUmbraco.Auth
+{
+    public abstract class UmbracoMemberAuthApiController : UmbracoApiController
+    {
+        /// <summary>
+        /// When a user has been authorised with a JWT Auth token
+        /// This is the member represents the authorised member
+        /// that can then be used in the API controller for Umbraco API Service Calls 
+        /// such as creating content
+        /// </summary>
+        public IMember AuthorisedMember
+        {
+            get
+            {
+                return ControllerContext.Request.Properties["umbraco-member"] as IMember;
```

Comment:
See my other comment about authentication and authorization, this shouldn't be necessary since this is a hack instead of using Claims in the current user/principal in the request. This whole controller should be able to be removed if Claims are used. I'll see what i can do